packages/integrations/src/media-organizer/lidarr/lidarr-integration.ts

```diff
@@ -7,9 +7,10 @@ import type { IntegrationTestingInput } from "../../base/integration";
 import { TestConnectionError } from "../../base/test-connection/test-connection-error";
 import type { TestingResult } from "../../base/test-connection/test-connection-service";
 import type { CalendarEvent } from "../../calendar-types";
+import type { CalendarIntegration } from "../../interfaces/calendar/calendar-integration";
 import { MediaOrganizerIntegration } from "../media-organizer-integration";
 
-export class LidarrIntegration extends MediaOrganizerIntegration {
+export class LidarrIntegration extends MediaOrganizerIntegration implements CalendarIntegration {
```

Comment:
Maybe instead still use `Integration` and provide array from `MediaOrganizerIntegration` somewhere else